packages/block-library/src/tag-cloud/index.php

```diff
@@ -29,8 +29,13 @@ function render_block_core_tag_cloud( $attributes ) {
 	);
 	$tag_cloud = wp_tag_cloud( $args );
 
-	if ( ! $tag_cloud ) {
```

Comment:
What's the reason for chaining `!` to `empty`?

Reply:
Makes code more readable in my opinion. Do you need me to revert it to avoid unnecessary changes?